src/components/forms/ProxyDeviceInput.tsx

```diff
@@ -0,0 +1,85 @@
+import { FC, useState } from "react";
+import { Input, Select } from "@canonical/react-components";
+
+interface Props {
+  onChange?: (type?: string, address?: string, port?: string) => void;
+  value?: string;
+  isConnect?: boolean;
+  isNatEnabled?: string;
```

Comment:
```suggestion
  isTypeDisabled,
```
nat is not a local concept to this component, I'd name this towards the local result of the variable.